### PR TITLE
build(prewhere): Change default prewhere settings

### PR DESCRIPTION
**Pull request by BYK**
Make the `PREWHERE_KEYS` default to what it is set everywhere else.

### CODE REVIEW

**Comment by tkaemming** [comment]
I'm +1 on this, but @JTCunning has the most context here probably, so I added him.

File: snuba/settings_base.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
 RETENTION_OVERRIDES = {}
 
 # the list of keys that will upgrade from a WHERE condition to a PREWHERE
-PREWHERE_KEYS = ['project_id']
+PREWHERE_KEYS = ['issue', 'tags[sentry:release]', 'message', 'environment']
```
Comment (JTCunning) [inline snuba/settings_base.py]: Please update this to the setting we currently use:
```
['event_id', 'issue', 'tags[sentry:release]', 'message', 'environment', 'project_id']
```

**Comment by BYK** [comment]
@JTCunning done, thanks :)

File: snuba/settings_base.py (hunk 1 above)
Comment (BYK) [reply snuba/settings_base.py]: Isn't `'tags[sentry:release]'` specific to sentry.io?

Comment (BYK) [reply snuba/settings_base.py]: (just realized this)

Comment (JTCunning) [reply snuba/settings_base.py]: No. It's the promoted release tag.

**Pull request merged**